lib/model.js

```diff
@@ -1885,6 +1885,29 @@ class Model {
     return this.findOne(options);
   }
 
+  /**
+ * Search for a many instances by its primary key._
+ *
+ * @param  {Array}      param The value of the desired instance's primary key.
+ *
+ * @see
+ * {@link Model.findAll}           for a full explanation of options, Note that options.where is not supported.
+ *
+ * @returns {Promise<Model>}
+ */
+  static findByPks(param) {
+  // return Promise resolved with null if no arguments are passed
+    if ([null, undefined].includes(param)) {
+      return Promise.resolve(null);
+    }
```

Comment:
Please remove this `if`, I think it's better to throw the error from the line below instead of silently resolving with null.